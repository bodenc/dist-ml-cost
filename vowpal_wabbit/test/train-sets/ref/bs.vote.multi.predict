0.000000 0.000000 0.000000
0.000000 0.000000 0.165033
0.000000 0.000000 0.148058
0.000000 0.000000 0.281402
0.000000 0.000000 0.287569
0.000000 0.000000 0.155553
0.000000 0.108261 0.323972
1.000000 0.334371 0.579987
0.000000 0.138097 0.312658
1.000000 0.491994 0.802263
0.000000 0.345480 0.409764
0.000000 0.296701 0.578739
1.000000 0.214216 0.691503
1.000000 0.404077 0.779290
0.000000 0.194051 0.467086
0.000000 0.447621 0.658849
1.000000 0.497381 0.776563
1.000000 0.632748 0.699377
1.000000 0.846292 1.078837
0.000000 0.256639 0.588491
0.000000 0.406493 0.635240
1.000000 0.534231 1.386442
1.000000 0.568045 0.855449
0.000000 0.434818 0.715776
0.000000 0.254717 0.352146
1.000000 0.537119 1.118137
0.000000 0.451506 0.644534
1.000000 0.560665 1.027926
1.000000 0.470868 0.605953
1.000000 0.566171 0.843057
1.000000 0.820930 0.985729
1.000000 0.333400 0.729816
1.000000 0.292014 0.746684
1.000000 0.835582 1.149324
1.000000 0.978270 1.115874
1.000000 0.523596 0.813854
1.000000 0.861985 1.434497
1.000000 0.538207 1.034530
1.000000 0.838307 1.290410
1.000000 0.898986 1.384085
0.000000 0.259948 0.521159
1.000000 0.565942 0.801054
1.000000 0.716282 1.174261
1.000000 0.607306 0.868094
1.000000 0.630561 0.840764
1.000000 0.505442 0.868526
1.000000 0.895612 1.736374
1.000000 1.071656 1.540534
1.000000 0.378841 0.873587
1.000000 0.444882 0.818013
1.000000 0.655278 1.085361
1.000000 0.772557 1.057850
1.000000 0.876981 1.166861
1.000000 0.851731 1.075140
1.000000 0.719158 0.948137
1.000000 1.123501 1.517079
1.000000 0.871863 1.291581
1.000000 0.612837 0.923135
1.000000 0.981842 1.529981
1.000000 0.806800 1.211975
1.000000 0.638449 0.914454
1.000000 0.712496 1.265780
1.000000 0.701968 0.990293
1.000000 0.434853 0.786216
1.000000 0.901428 1.349504
1.000000 0.619762 1.089103
0.000000 0.370273 0.818540
1.000000 0.495578 1.288672
1.000000 0.527644 0.927179
1.000000 0.467431 0.842085
1.000000 0.593316 1.297979
1.000000 0.671109 1.149071
1.000000 0.558884 0.917177
0.000000 0.459957 1.020855
0.000000 0.264029 0.545274
0.000000 0.121569 0.419695
1.000000 0.430444 0.917727
1.000000 0.558016 0.838422
0.000000 0.193284 0.440836
1.000000 0.708393 0.924531
0.000000 0.134463 0.485917
0.000000 0.243330 0.558989
1.000000 0.104232 0.849344
1.000000 0.554639 0.683434
1.000000 0.334042 0.533729
0.000000 0.279631 0.379622
0.000000 0.102734 0.288614
1.000000 0.573906 0.886307
1.000000 0.468160 1.164194
0.000000 -0.003624 0.601272
0.000000 0.162286 0.314017
0.000000 0.122719 0.432552
1.000000 0.432279 0.902288
0.000000 0.204582 0.378572
0.000000 0.207140 0.483184
0.000000 0.238760 0.616861
0.000000 0.430436 0.662567
1.000000 0.592202 0.701785
0.000000 0.255310 0.558475
0.000000 0.080374 0.499227
1.000000 0.731459 1.042138
0.000000 0.217919 0.603396
0.000000 0.026271 0.584494
1.000000 0.623264 0.803858
0.000000 0.162409 0.530250
1.000000 0.518258 0.690469
1.000000 0.411061 0.745736
0.000000 0.036538 0.340745
0.000000 0.269086 0.387784
0.000000 0.164244 0.548558
1.000000 0.483396 0.984191
0.000000 0.326028 0.593281
0.000000 0.373174 0.442815
1.000000 0.574639 0.831625
0.000000 0.189635 0.480897
0.000000 0.143763 0.388238
0.000000 0.227446 0.527868
1.000000 0.223253 0.589372
0.000000 0.271011 0.414272
0.000000 0.255855 0.497019
0.000000 0.430396 0.584152
1.000000 0.564957 0.605265
0.000000 0.358504 0.750753
1.000000 0.561140 1.055806
1.000000 0.657542 0.811234
1.000000 0.301204 0.526818
1.000000 0.500536 0.637776
1.000000 0.173054 0.582644
1.000000 0.469879 0.863764
1.000000 0.290989 0.581895
0.000000 0.209859 0.471660
0.000000 0.301568 0.630369
0.000000 0.262337 0.508351
0.000000 0.290912 0.439717
0.000000 0.454644 0.528507
1.000000 0.387631 0.775304
1.000000 0.658072 0.884307
1.000000 0.394046 0.556557
0.000000 0.067347 0.546341
1.000000 0.457214 0.777604
1.000000 0.081713 0.990856
0.000000 0.123409 0.431178
1.000000 0.558038 0.839650
1.000000 0.509381 0.857204
1.000000 0.758660 0.923783
0.000000 0.203302 0.621093
1.000000 0.257793 0.701938
1.000000 0.684787 1.296125
1.000000 0.546581 0.715700
1.000000 0.609479 0.977906
0.000000 0.343596 0.613114
1.000000 0.453565 0.595525
1.000000 0.699039 0.872309
1.000000 0.802597 1.131345
1.000000 0.773064 0.882122
1.000000 0.679996 0.790733
1.000000 0.821519 1.206804
1.000000 0.876227 1.184705
1.000000 0.638430 0.860349
1.000000 0.892868 0.989506
1.000000 0.663904 0.914616
1.000000 0.611629 1.085364
1.000000 1.034016 1.378084
1.000000 0.575567 1.310108
1.000000 0.528280 0.752936
1.000000 0.387198 0.713847
1.000000 0.847409 1.384879
1.000000 0.710114 1.046122
1.000000 0.448540 0.882891
1.000000 0.501730 1.292727
1.000000 0.753089 1.088482
1.000000 0.428781 0.851308
1.000000 0.056200 0.677738
1.000000 0.465845 0.578409
1.000000 0.326192 0.758562
1.000000 0.344810 0.960967
0.000000 0.193242 0.613759
1.000000 0.640149 1.154629
1.000000 0.511056 0.717236
1.000000 0.387025 1.176268
1.000000 0.650396 0.912986
1.000000 0.375709 0.758377
1.000000 0.417755 0.670758
0.000000 0.034128 0.396980
1.000000 0.141161 0.701032
1.000000 0.612369 0.808358
1.000000 0.680022 0.936695
1.000000 0.710665 1.276183
1.000000 0.280899 0.678387
1.000000 0.685258 0.910630
1.000000 0.451684 1.083901
0.000000 -0.189737 0.464664
0.000000 0.335778 0.487427
0.000000 0.354272 0.608350
0.000000 0.216469 0.569561
1.000000 0.664424 0.937707
1.000000 0.528500 0.682091
1.000000 0.396934 0.676104
0.000000 0.314606 0.486035
1.000000 0.423501 0.815634
1.000000 0.599465 1.198681
1.000000 0.444006 0.745522
2.000000 0.217497 2.305237
0.000000 0.009720 0.610486
0.000000 0.132152 0.513686
2.000000 1.439295 1.736483
0.000000 0.098901 0.182703
0.000000 -0.019331 0.510164
1.000000 0.550836 2.560719
0.000000 0.453664 1.021923
1.000000 0.427721 0.816875
2.000000 0.341128 2.003870
0.000000 0.278227 0.848862
0.000000 0.303663 0.706136
1.000000 1.230327 1.625183
0.000000 0.328733 0.848609
1.000000 0.794979 1.330024
1.000000 0.205459 1.745439
1.000000 0.467915 1.162204
1.000000 0.305884 0.607911
1.000000 0.585154 1.564143
1.000000 1.012043 1.537921
0.000000 0.031999 0.738372
2.000000 0.634158 1.955908
0.000000 -0.298638 0.177263
1.000000 0.189603 0.706815
2.000000 1.048168 2.650965
0.000000 -0.236488 0.408123
1.000000 0.576108 0.928061
1.000000 0.618888 1.045780
1.000000 0.768219 1.103604
1.000000 0.444926 1.019207
2.000000 2.307381 2.666956
0.000000 0.222740 0.650724
1.000000 0.851335 1.387224
1.000000 0.980378 2.346185
1.000000 0.524254 1.307210
1.000000 -0.255680 0.730805
2.000000 0.928889 2.563099
1.000000 1.106943 1.543448
0.000000 0.147832 0.499101
2.000000 1.713146 1.922755
0.000000 -0.422890 0.276687
1.000000 0.137631 0.818028
0.000000 0.483791 1.990389
0.000000 0.130376 0.680534
1.000000 0.380894 1.364457
2.000000 1.520614 2.139831
1.000000 0.467767 1.301225
1.000000 0.556047 0.857015
2.000000 0.807443 2.242766
1.000000 0.422238 1.071977
2.000000 0.548377 2.387461
0.000000 -0.235260 0.950785
1.000000 0.812233 1.445712
1.000000 1.371125 2.260124
0.000000 -0.102339 0.366287
1.000000 0.343520 2.032098
1.000000 0.880926 1.480993
1.000000 0.691308 1.124617
0.000000 0.071102 0.985121
0.000000 0.140805 1.390547
1.000000 0.812553 0.958269
0.000000 -0.043721 0.891116
0.000000 -0.223033 1.078083
0.000000 -0.051314 1.159963
0.000000 0.018502 0.577511
0.000000 -0.492692 1.194983
1.000000 0.515916 0.646594
1.000000 0.742872 1.031669
0.000000 -0.172807 0.998196
1.000000 1.084413 1.247002
1.000000 0.738994 0.929806
0.000000 0.242825 0.946261
1.000000 0.246485 1.118194
1.000000 0.396957 1.106297
0.000000 0.099268 0.424030
1.000000 0.341410 1.104251
0.000000 0.084095 0.256575
1.000000 0.734654 1.226992
0.000000 0.093251 0.322649
0.000000 0.083004 0.597996
1.000000 0.830856 1.354751
1.000000 -0.444640 0.981413
0.000000 -0.012969 0.325183
0.000000 0.038056 0.466260
0.000000 0.235077 1.067551
1.000000 0.822835 1.163692
0.000000 -0.307307 0.947464
1.000000 0.407984 0.981901
0.000000 0.105088 0.191917
1.000000 0.591428 0.925356
1.000000 0.912386 1.379315
0.000000 0.045923 0.242303
1.000000 -0.360054 0.803472
0.000000 0.076753 0.487522
1.000000 0.833233 1.334028
1.000000 0.129056 0.860489
1.000000 0.836070 1.270873
1.000000 0.393782 0.554894
0.000000 0.225144 1.224271
1.000000 0.551031 0.920157
0.000000 0.187117 0.678324
0.000000 -0.275916 0.998000
0.000000 0.125318 0.426210
1.000000 0.882179 1.057088
0.000000 -0.128260 0.151221
0.000000 -0.131467 0.192410
2.000000 0.438792 2.542680
1.000000 0.339022 0.666365
1.000000 0.585576 0.959958
1.000000 0.662711 1.736787
0.000000 0.082317 0.579463
1.000000 0.941888 1.216104
0.000000 0.223651 0.483687
0.000000 -0.097679 0.541966
1.000000 0.526698 1.322495
1.000000 0.599471 0.930260
1.000000 0.212480 1.090623
0.000000 0.357562 0.683488
0.000000 -0.072195 0.532846
1.000000 0.921899 1.075981
2.000000 0.549564 1.968571
1.000000 0.762271 1.120742
1.000000 0.820842 1.069257
0.000000 0.007353 0.707822
0.000000 0.007147 0.594827
1.000000 -0.059258 0.754897
0.000000 -0.178675 0.841736
1.000000 0.324514 0.764336
0.000000 -0.064208 0.215662
1.000000 0.420925 0.985930
0.000000 0.050083 0.452892
0.000000 0.348309 1.144155
2.000000 0.519452 2.069813
1.000000 0.019529 0.622346
1.000000 0.627421 1.114256
0.000000 0.190601 0.832675
1.000000 0.398156 1.261313
0.000000 0.258800 0.483903
1.000000 0.031918 0.748525
0.000000 -0.097232 0.190355
1.000000 0.493372 1.001214
1.000000 0.483948 1.637048
1.000000 0.673953 1.067169
1.000000 -0.026007 0.934958
1.000000 0.904239 1.243806
1.000000 0.824110 2.304790
0.000000 0.137077 0.781272
1.000000 0.803111 1.054000
1.000000 0.309516 0.951626
0.000000 0.413858 2.864440
1.000000 0.568128 0.924083
0.000000 0.350749 0.948557
1.000000 0.708602 2.655971
1.000000 0.683605 1.079867
1.000000 0.958679 1.203365
3.000000 2.486275 2.828902
1.000000 0.288297 1.169050
1.000000 0.833058 1.039771
1.000000 0.305114 1.133428
0.000000 -0.110090 0.659091
1.000000 -0.136350 1.224722
1.000000 0.193338 1.516059
0.000000 0.258221 0.711354
1.000000 0.562552 1.107565
1.000000 1.061759 1.296170
1.000000 0.395156 0.767103
1.000000 -0.726840 0.794076
1.000000 0.612856 1.229840
1.000000 0.870214 1.331301
1.000000 0.139298 0.796273
0.000000 -0.236868 0.032399
0.000000 0.324254 0.820511
0.000000 0.340607 0.760077
0.000000 -0.221366 0.642247
0.000000 -0.598052 0.316288
1.000000 0.626963 1.123345
1.000000 0.648284 1.167557
1.000000 0.615433 0.812082
1.000000 0.951146 1.156022
1.000000 0.312134 0.710594
1.000000 0.610337 0.918649
0.000000 -0.035791 0.452970
0.000000 -0.101782 0.610013
-1.000000 -0.791304 0.636218
1.000000 0.679698 0.942629
1.000000 0.288574 1.229743
0.000000 -0.136277 0.574833
1.000000 0.508833 1.146723
1.000000 0.655831 1.035127
0.000000 -0.031817 0.435649
0.000000 -0.080079 0.113897
0.000000 -0.065801 0.639572
0.000000 0.099980 0.445024
1.000000 0.889922 1.031209
1.000000 0.647491 1.101244
0.000000 0.014271 0.156800
0.000000 0.159439 0.484418
1.000000 0.302375 0.897387
1.000000 1.081031 1.199342
1.000000 0.202734 0.856565
2.000000 0.339812 2.417229
0.000000 -0.106936 0.491367
0.000000 -0.080034 0.438243
2.000000 1.744303 2.270882
0.000000 -0.007449 0.252462
0.000000 0.078601 0.363655
3.000000 2.328100 2.989328
1.000000 0.434498 0.741918
0.000000 0.232496 0.601733
3.000000 0.342625 2.731194
1.000000 0.068349 0.673318
0.000000 0.185838 0.366195
2.000000 1.296891 2.146153
1.000000 0.892761 1.072770
1.000000 0.987497 1.049127
2.000000 1.706062 2.331514
0.000000 0.143616 1.117379
0.000000 0.076147 0.482104
2.000000 0.588055 2.283137
1.000000 0.995039 1.438469
0.000000 -0.042263 0.758128
2.000000 1.866143 2.414660
0.000000 -0.291775 0.098181
0.000000 0.093025 0.396851
3.000000 2.434493 2.786772
0.000000 -0.151037 0.198047
1.000000 0.556628 0.920802
1.000000 0.581485 0.894279
1.000000 0.983617 1.200979
0.000000 0.184066 0.633886
3.000000 2.205104 2.909763
0.000000 0.050778 0.234949
1.000000 0.783287 0.998630
3.000000 2.254206 3.000000
1.000000 0.833646 1.149665
0.000000 -0.171880 0.526677
2.000000 0.847912 2.631598
1.000000 1.313866 1.546921
0.000000 -0.004341 0.143046
2.000000 1.632723 2.221723
0.000000 -0.228894 0.118567
1.000000 0.088834 0.860135
1.000000 0.514293 2.492078
0.000000 -0.040217 0.505429
1.000000 0.133565 0.834474
2.000000 1.793768 2.450782
1.000000 0.846066 1.021544
1.000000 0.644885 0.874056
2.000000 0.763899 2.692840
1.000000 0.557813 1.005416
3.000000 0.276801 2.979913
0.000000 0.004172 0.718604
1.000000 1.376880 1.904127
2.000000 2.380367 2.923159
0.000000 -0.141535 -0.020126
3.000000 0.371392 2.613635
1.000000 0.852091 1.257032
1.000000 0.787321 0.963984
0.000000 -0.463310 0.869703
0.000000 -0.126471 0.365185
1.000000 0.756597 1.026044
0.000000 -0.056285 0.176707
0.000000 -0.729904 0.105145
0.000000 -0.192329 0.642350
0.000000 0.018216 0.152067
-1.000000 -0.785591 0.691261
0.000000 0.233399 0.462416
1.000000 0.784114 1.021737
0.000000 -0.360368 -0.088223
1.000000 1.142311 1.313959
1.000000 0.717487 1.050908
0.000000 0.285390 0.484370
1.000000 0.526287 0.980845
1.000000 0.550255 1.131724
0.000000 -0.000358 0.171991
0.000000 0.152881 0.829533
0.000000 -0.005759 0.158383
1.000000 0.703276 1.141608
0.000000 -0.225007 0.336938
0.000000 -0.007622 0.381598
1.000000 0.961659 1.576711
-1.000000 -0.714439 -0.434934
0.000000 -0.145412 0.168885
0.000000 -0.050687 0.114173
1.000000 0.436958 1.208459
1.000000 0.843157 1.062499
0.000000 -0.757360 0.037304
1.000000 0.635665 0.898244
0.000000 -0.041465 0.153270
1.000000 0.871555 1.035236
1.000000 0.917454 1.304591
0.000000 -0.040445 0.326092
0.000000 -0.727551 -0.120701
0.000000 0.053472 0.601203
1.000000 0.876527 1.395881
0.000000 0.136357 0.929924
1.000000 0.865032 1.332113
1.000000 0.491876 0.767683
0.000000 0.084244 0.214354
1.000000 0.631268 1.284191
0.000000 -0.018549 1.065585
-1.000000 -0.887425 -0.175965
0.000000 -0.015399 0.308129
1.000000 1.010721 1.280371
0.000000 -0.065664 0.241675
0.000000 -0.001636 0.277354
3.000000 0.444786 2.945367
1.000000 0.558017 0.764378
1.000000 0.757813 1.289041
2.000000 0.918353 2.354488
0.000000 0.130477 0.516690
1.000000 0.975055 1.281581
0.000000 0.290821 0.567054
0.000000 -0.027672 0.430186
2.000000 0.484094 1.775637
1.000000 0.956242 1.445309
1.000000 0.816529 1.083200
0.000000 0.448013 0.671826
0.000000 -0.022111 0.244937
1.000000 1.025617 1.182287
2.000000 0.633402 2.825060
1.000000 0.989935 1.260734
1.000000 1.026750 1.178187
1.000000 0.476786 1.011950
0.000000 -0.047676 0.132949
1.000000 0.304056 0.996979
0.000000 0.164144 0.710530
1.000000 0.391962 0.881798
0.000000 0.072250 0.185691
1.000000 0.933522 1.107188
0.000000 0.111519 0.549682
1.000000 0.916077 1.075142
2.000000 0.508768 2.854694
0.000000 0.186887 0.634426
1.000000 0.997211 1.145699
1.000000 0.904702 1.130843
1.000000 0.889082 1.331963
0.000000 0.345047 0.521201
0.000000 0.222160 0.452721
0.000000 -0.032865 0.164988
1.000000 1.100104 1.270337
2.000000 1.360059 2.173015
1.000000 0.878364 1.068048
1.000000 0.524773 0.908153
1.000000 0.916697 1.317240
2.000000 1.545921 2.654551
0.000000 0.242684 0.714910
1.000000 0.994405 1.139611
1.000000 0.144237 0.911305
3.000000 2.884569 3.000000
1.000000 0.644231 1.025839
1.000000 0.380085 1.022635
3.000000 2.383001 2.881700
1.000000 0.789394 0.957005
1.000000 0.971576 1.160997
3.000000 2.425505 2.953607
1.000000 0.575379 1.154130
1.000000 0.887880 1.024900
0.000000 0.285230 1.041253
0.000000 -0.158340 0.166777
1.000000 0.057619 1.124275
1.000000 0.158257 1.191980
0.000000 0.133091 0.634240
1.000000 0.602931 1.034091
1.000000 0.792930 1.094189
0.000000 0.316140 0.768163
-1.000000 -0.957332 -0.745123
1.000000 0.961031 1.168193
1.000000 0.941535 1.132468
0.000000 0.030088 0.189962
0.000000 -0.140340 0.096641
1.000000 0.389097 0.900519
0.000000 0.066681 0.798181
0.000000 -0.277573 0.255976
-1.000000 -0.773447 0.185097
1.000000 0.734497 1.132766
1.000000 1.002565 1.166083
1.000000 0.670390 0.763246
1.000000 1.001861 1.243103
0.000000 0.290797 0.650596
1.000000 0.793791 0.989055
0.000000 -0.074787 0.336271
0.000000 -0.074658 0.133973
-1.000000 -0.862773 -0.621745
1.000000 0.847320 1.029697
1.000000 0.192357 0.682400
0.000000 -0.133584 0.290000
1.000000 0.473183 1.052054
1.000000 0.792653 1.087659
0.000000 -0.032790 0.423253
0.000000 -0.091083 0.074520
0.000000 -0.137861 -0.020267
0.000000 0.026626 0.155992
1.000000 0.935695 1.240415
1.000000 0.886110 1.103984
0.000000 -0.029820 0.316957
0.000000 0.032081 0.680008
1.000000 0.448572 0.873944
