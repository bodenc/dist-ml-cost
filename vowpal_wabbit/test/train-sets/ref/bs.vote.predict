0.000000 0.000000 0.000000
0.000000 0.000000 0.165033
0.000000 0.000000 0.148058
0.000000 0.000000 0.056914
0.000000 0.000000 0.055368
0.000000 0.000000 0.108588
0.000000 0.060035 0.099882
0.000000 0.185384 0.214994
0.000000 0.076365 0.133533
0.000000 0.153715 0.229805
0.000000 0.151824 0.281808
0.000000 0.124296 0.267574
0.000000 0.052844 0.236524
0.000000 0.146848 0.297702
0.000000 0.095344 0.189743
0.000000 0.263105 0.362490
0.000000 0.188637 0.294482
0.000000 0.314464 0.363651
0.000000 0.391540 0.490716
0.000000 0.129572 0.261093
0.000000 0.069807 0.134286
0.000000 0.280419 0.559181
0.000000 0.203773 0.384708
0.000000 0.212224 0.261401
0.000000 0.110674 0.136587
0.000000 0.292162 0.489801
0.000000 0.138171 0.244588
0.000000 0.214734 0.363004
0.000000 0.244205 0.385861
0.000000 0.231640 0.377310
0.000000 0.346767 0.538611
0.000000 0.116643 0.256667
0.000000 0.135187 0.213753
0.000000 0.266950 0.357525
0.000000 0.326916 0.541221
0.000000 0.232655 0.516905
0.000000 0.223680 0.378183
0.000000 0.279042 0.426095
0.000000 0.358019 0.456708
1.000000 0.444014 0.563562
0.000000 0.208084 0.394655
0.000000 0.265202 0.434631
0.000000 0.330610 0.459316
0.000000 0.204864 0.334043
0.000000 0.264325 0.352090
0.000000 0.122603 0.278541
0.000000 0.337852 0.663552
0.000000 0.389132 0.541451
0.000000 0.065278 0.283866
0.000000 0.137934 0.193465
0.000000 0.223702 0.296520
0.000000 0.273999 0.323600
0.000000 0.339832 0.371714
0.000000 0.293943 0.427828
0.000000 0.427890 0.529465
1.000000 0.535167 0.703842
0.000000 0.393061 0.459777
0.000000 0.255470 0.342168
0.000000 0.382347 0.476401
0.000000 0.279962 0.380802
0.000000 0.266670 0.319516
0.000000 0.354260 0.441987
1.000000 0.400923 0.619577
0.000000 0.178755 0.332358
0.000000 0.434999 0.491165
0.000000 0.225289 0.315372
0.000000 0.300479 0.432123
1.000000 0.609933 0.803870
0.000000 0.305149 0.435861
0.000000 0.348280 0.558524
0.000000 0.382903 0.416826
0.000000 0.409965 0.560847
0.000000 0.181245 0.388152
0.000000 0.206158 0.387712
0.000000 0.206952 0.514329
0.000000 0.258637 0.408113
0.000000 0.175173 0.466962
1.000000 0.328643 0.587575
0.000000 0.125153 0.153853
0.000000 0.140399 0.238694
0.000000 0.210751 0.259890
0.000000 0.220530 0.347812
1.000000 0.537810 0.595470
0.000000 0.246706 0.371924
0.000000 0.167971 0.435135
0.000000 0.231589 0.307771
0.000000 0.214186 0.235897
1.000000 0.368226 0.654591
1.000000 0.539059 0.683148
0.000000 0.200537 0.395829
0.000000 0.072891 0.139037
0.000000 0.255444 0.503878
1.000000 0.718731 0.908420
0.000000 0.148134 0.234985
1.000000 0.419400 0.526940
0.000000 0.281590 0.449685
0.000000 0.416415 0.524149
0.000000 0.181038 0.257179
0.000000 0.288000 0.435358
0.000000 0.241666 0.383677
1.000000 0.542362 0.815245
1.000000 0.406909 0.555072
0.000000 0.307712 0.434762
0.000000 0.413690 0.456132
1.000000 0.394815 0.766386
0.000000 0.465605 0.558527
0.000000 0.123853 0.193798
0.000000 0.322638 0.425156
0.000000 0.152502 0.277716
1.000000 0.302211 0.603111
1.000000 0.519083 0.691395
1.000000 0.299794 0.647601
0.000000 0.384462 0.517522
0.000000 0.221309 0.305192
0.000000 0.367274 0.503765
0.000000 0.337746 0.591077
0.000000 0.145899 0.263625
1.000000 0.414073 0.599591
0.000000 0.152904 0.557253
0.000000 0.345800 0.410097
0.000000 0.283921 0.442342
0.000000 0.420805 0.540704
0.000000 0.382728 0.467405
0.000000 0.332877 0.487902
1.000000 0.483051 0.669946
0.000000 0.311442 0.509110
0.000000 0.304542 0.405824
1.000000 0.476058 0.722197
0.000000 0.087010 0.356679
0.000000 0.286310 0.554956
0.000000 0.262545 0.395812
1.000000 0.456266 0.832514
0.000000 0.162775 0.234763
0.000000 0.237567 0.308066
0.000000 0.304806 0.417123
0.000000 0.187233 0.322587
1.000000 0.509059 0.586988
0.000000 0.337408 0.492913
1.000000 0.637149 0.806485
0.000000 0.290388 0.402845
0.000000 0.409257 0.463213
0.000000 0.205731 0.290107
1.000000 0.703565 0.780184
1.000000 0.762320 0.883820
0.000000 0.430970 0.536012
1.000000 0.453806 0.627823
1.000000 0.421150 0.595644
1.000000 0.778521 0.896804
0.000000 0.263792 0.487992
0.000000 0.314233 0.442666
1.000000 0.340442 0.744408
0.000000 0.265697 0.386746
1.000000 0.588076 0.721545
0.000000 0.476271 0.573887
1.000000 0.537259 0.752284
1.000000 0.540044 0.642390
1.000000 0.789948 0.897157
1.000000 0.499518 0.624615
1.000000 0.674253 0.872026
1.000000 0.439344 0.610095
1.000000 0.407931 0.684587
0.000000 0.227526 0.417340
0.000000 0.245463 0.601820
1.000000 0.647145 0.707758
0.000000 0.292232 0.355189
0.000000 0.394226 0.478338
1.000000 0.729621 0.905942
0.000000 0.326497 0.618210
1.000000 0.424470 0.527758
1.000000 0.398567 0.780571
1.000000 0.644528 0.725458
0.000000 0.263530 0.378992
0.000000 0.215015 0.308514
0.000000 0.317181 0.408162
0.000000 0.124465 0.178991
1.000000 0.467243 0.638149
0.000000 0.281433 0.327047
1.000000 0.779584 0.985479
1.000000 0.539432 0.687287
1.000000 0.698828 0.777052
0.000000 0.396852 0.521239
0.000000 0.361414 0.418624
1.000000 0.443891 0.590845
0.000000 0.278819 0.443288
0.000000 0.163261 0.431671
0.000000 0.279712 0.564609
1.000000 0.600835 0.827839
1.000000 0.534030 0.688263
0.000000 0.198917 0.376790
1.000000 0.510130 0.786455
1.000000 0.450644 0.712191
0.000000 0.088898 0.485186
0.000000 0.209145 0.319709
0.000000 0.260771 0.315615
0.000000 0.256404 0.383797
0.000000 0.454846 0.706750
0.000000 0.446069 0.578452
0.000000 0.282540 0.459406
0.000000 0.254622 0.458421
1.000000 0.386176 0.671750
