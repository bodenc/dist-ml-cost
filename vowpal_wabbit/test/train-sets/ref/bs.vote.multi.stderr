final_regressor = models/bs.vote.multi.model
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
decay_learning_rate = 1
predictions = bs.vote.multi.predict
creating cache_file = train-sets/0001.multi.dat.cache
Reading datafile = train-sets/0001.multi.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
1.000000   1.000000            1         1.0   1.0000   0.0000       51
0.500000   0.000000            2         2.0   0.0000   0.0000      104
0.500000   0.500000            4         4.0   0.0000   0.0000      135
0.500000   0.500000            8         8.0   0.0000   1.0000      146
0.625000   0.750000           16        16.0   1.0000   1.0000      143
0.656250   0.687500           32        32.0   1.0000   1.0000       70
0.703125   0.750000           64        64.0  -1.0000   1.0000       34
0.585938   0.468750          128       128.0   0.0000   0.0000       30
0.535714   0.535714          256       256.0  -1.0000   1.0000       72 h
0.482143   0.428571          512       512.0   0.0000   0.0000       37 h

finished run
number of examples per pass = 180
passes used = 3
weighted example sum = 540
weighted label sum = 0
average loss = 0.4 h
best constant = 0
total feature number = 41349
